release-pipeline.yml

```diff
@@ -860,6 +962,13 @@ stages:
               archiveFile: '$(archives)/azcopy_windows_386_$(azcopy_version).zip'
             continueOnError: true
 
+          - task: ArchiveFiles@2
+            displayName: 'Archive Windows 32 bit Build'
+            inputs:
+              rootFolderOrFile: '$(windows_386)'
+              archiveFile: '$(archives)/azcopy_windows_386_$(azcopy_version).zip'
```

Comment:
Rename the zip file to azcopy_windows_grpc_386_$(azcopy_version).zip' as azcopy_windows_386_$(azcopy_version).zip already exists

Reply:
Nice catch! Thank you.